docs/source/draft/models.rst

```diff
@@ -0,0 +1,89 @@
+.. _draft_models:
+
+Draft Models
+============
+
+The following database models are being considered in the implementing IAM:
+
+.. contents::
+
+.. note::
+
+    All fields are 'Unique' within their given table.
+
+
+Clients
+--------
+
+Clients are stored in the database with the given schema.
+
+========== ======== ====== ====================================================
+Field      Type     Unique Description
+---------- -------- ------ ----------------------------------------------------
+client_id  integer  true   Client's Unique ID.
+name       string   true   Client's human readable name.
+========== ======== ====== ====================================================
+
+
+Projects
+--------
+
+Projects are stored in the database with the following schema:
+
+============= ================ ====== =========================================
+Field         Type             Unique Description
+------------- ---------------- ------ -----------------------------------------
+project_id    integer          true   Project's unique ID.
+project_name  string           true   Project's human readable name.
+client_id     foreign key      false  Client which owns this project.
+resources     serialized list  false  For example: "cpu,ftp,db"
+============= ================ ====== =========================================
+
+
+Resources
+---------
+
+A given table for ``resourceX`` (e.g., ``node``, ``DB``, ``FTP``) will have the
+following fields of the given type:
+
+=========================== =========== ======= =================================
+Field                       Type        Unique  Description
+--------------------------- ----------- ------- ---------------------------------
+resourceX_id                integer     true    Resources unique ID.
+project_id                  foreign key false   Project which owns this resource.
+resources_specific_field_1  var         false   Any specific metadata about
+resources_specific_field_2  var         false   |
+...etc                      var         false   |
+=========================== =========== ======= =================================
+
+
+Measurements
+------------
+
+A given table for ``measurementX`` (e.g., ``CPU``, ``Memory``, ``Bandwidth``)
+will have the following fields of the given type:
+
+=============== ============ ======= ===================================================
+Field           Type         Unique  Description
+--------------- ------------ ------- ---------------------------------------------------
+measurementX_id integer      true    Measurement unique id.
```

Comment:
Again, this is really just the standard id for this table, no need for measurement_id